README.md

```diff
@@ -212,4 +213,10 @@ After the Flexy Java application component has been installed and started, the c
 Source code and an Eclipse project for the Flexy Java app are made available in the [hms-networks/flexy-thingworx-connector](https://github.com/hms-networks/flexy-thingworx-connector) repository on GitHub. It is also included in the /source-flexy-java-app/ folder of Flexy Thingworx Connector release \(.zip\) files.
 
 #### Development Environment
-The Flexy Java application component was developed using a standard Ewon Java development environment. Documentation and additional information about the Ewon Java development environment is available in the Ewon Java Toolkit User Guide \(J2SE\) at [https://developer.ewon.biz/content/java-0#dev-documents](https://developer.ewon.biz/content/java-0#dev-documents). 
\ No newline at end of file
+The Flexy Java application component was developed using a standard Ewon Java development environment. Documentation and additional information about the Ewon Java development environment is available in the Ewon Java Toolkit User Guide \(J2SE\) at [https://developer.ewon.biz/content/java-0#dev-documents](https://developer.ewon.biz/content/java-0#dev-documents).
+
+## Javadocs
+Developer documentation is available in Javadoc format, can be generated by the following bash command: 
+```bash
+javadoc javadoc -private -splitindex -use -author -version -d docs -classpath  $(find src/ -name "*.java")
```

Comment:
I do think removing the Javadocs is the best idea, but in that case, I think we need to either provide them as part of releases or as seen here, a command to generate them. If we provide a command to users though, I think it needs to contain a Windows (Powershell or CMD) command as the primary, and any Linux/Unix bash commands should be secondary since many users run Windows.

Reply:
@it-hms if you wanted to include them in a release (via github actions) you should be able to adapt this action easily.
https://github.com/hms-networks/flexy-aws-connector/blob/master/.github/workflows/aws-release-built.yml